GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -73,6 +35,51 @@ gdjs.HowlerSound = class HowlerSound {
     this._paused = false;
     this._stopped = false;
     this._id = this._howl.play(this._id === null ? undefined : this._id);
+
+    if (!this._hasListeners) {
+      this._hasListeners = true;
+      /*
+       * Track events to be sure the status is
+       * sync'ed with the sound - though some should be redundant
+       * with `play`/`pause` methods already doing that. Keeping
+       * that to be sure that the status is always correct.
+       *
+       * This is done in play as we need the ID to set up those callbacks.
+       */
+
+      this.on('play', function () {
```

Comment:
This is still correct, you listen to the events of all the sounds of the same Howl, so you'll improperly set _paused/_stopped and other stuff for all sounds, which could create hard to debug issues.

You must either set your event only on the id of the Howl (in which case you need to know the id) OR you must read the id that is passed, according to what I see in Howler.js source code, as the first parameter of all callbacks.
If this id is not the same as `this._id`, it means that the callback is called for a sound that is not this one (either this one has a different id, or id was not yet set. In both cases it's not the same sound).

In all cases, triple check this because this seems to be a recipe for subtle bugs :)

Reply:
> You must either set your event only on the id of the Howl (in which case you need to know the id)

This is what I currently do? Note that I don't use thi._howl.on but this.on which passes the correct id. And I have the ID for sure, it just vot generated and stored.

I also tested and pausing doesn't pause other sounds or stuff like that currently.

Reply:
Ah right! I missed that you're using `this.on[ce]`. All good then, my bad I overlooked this 👍